src/website/src/components/SplashPage/SplashPage.jsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 // Import resources
-import ECLogo from '@ecl/preset-ec-core/dist/images/logo/logo--en.svg';
-import EULogo from '@ecl/preset-eu-core/dist/images/logo/logo--en.svg';
+import ECLogo from '@ecl/preset-ec-core/dist/images/logo/logo-ec--en.svg';
+import EULogo from '@ecl/preset-eu-core/dist/images/logo/standard-version/positive/logo-eu--en.svg';
```

Comment:
It all look consistent, you maybe even overdid it, changing also the "mocked" paths in the spec files, but i guess that came out of a find & replace.
(it's irrelevant but a few of the common (ec/eu) specs files for the site-headers have the "ec" in their name and this made notice that in the EU demos we are showing the "european commission" alt attribute, we can work in a separate tickets to fix this, in case..)
Just this, i guess, changed the size of the logo in the website splash page:
![image](https://user-images.githubusercontent.com/14326885/106158175-26405c80-618c-11eb-9d95-9bd32deb4846.png)